search-commons/src/main/java/no/unit/nva/utils/FramedJsonGenerator.java

```diff
@@ -0,0 +1,62 @@
+package no.unit.nva.utils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.jsonldjava.core.JsonLdOptions;
+import com.github.jsonldjava.core.JsonLdProcessor;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.JsonUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+@JacocoGenerated
+public class FramedJsonGenerator {
+
+    public static final ObjectMapper mapper = JsonUtils.objectMapper;
+    public static final String JSON_LD_GRAPH = "@graph";
+    private final Map<String, Object> framedJson;
+
+    public FramedJsonGenerator(List<InputStream> streams, InputStream frame) {
+        Map<?, ?> frameMap = null;
+        try {
+            frameMap = mapper.readValue(frame, Map.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        framedJson = JsonLdProcessor.frame(createGraphDocumentFromInputStreams(streams),
+                Objects.requireNonNull(frameMap), getDefaultOptions());
+    }
+
+    private Map<String, Object> createGraphDocumentFromInputStreams(List<InputStream> streams) {
+        ObjectNode document = mapper.createObjectNode();
+        ArrayNode graph = mapper.createArrayNode();
+        for (InputStream stream : streams) {
+            try (stream) {
+                JsonNode node = mapper.readTree(stream);
+                graph.add(node);
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
Why printing of stack trace. 
Suggestion:
```
 private Map<String, Object> createGraphDocumentFromInputStreams(List<InputStream> streams) {
        ObjectNode document = mapper.createObjectNode();
        ArrayNode graph = mapper.createArrayNode();
        streams.stream()
            .map(attempt(mapper::readTree))
            .filter(this::keepSuccessesAndLogErrors)
            .map(Try::orElseThrow)
            .forEach(graph::add);
        
        document.set(JSON_LD_GRAPH, graph);
        return mapper.convertValue(document, new TypeReference<>() {
        });
    }
```

Reply:
and refactored this method  also